docs/components/connectors/custom-built-connectors/connector-templates.md

```diff
@@ -442,6 +442,566 @@ a simple JSON configuration:
 </TabItem>
 </Tabs>
 
+## Inbound boundary event Connector templates
+
+You can, for example, allow the user to model and configure the following **HTTP webhook Connector** for boundary events by providing
+a simple JSON configuration:
+
+<Tabs groupId="connectorTemplateInbound" defaultValue="process" values={
+[
+{label: 'Process modeling view', value: 'process' },
+{label: 'Connector template configuration', value: 'json' }
+]
+}>
+
+<TabItem value='process'>
+
+![Webhook Inbound boundary Connector Example.png](img/custom-connector-template-inbound-boundary.png)
```

Comment:
The example is a little bit unclear as it doesnt contain any labels. I would also recommend to only use a single connector in the example. The one that is used for the boundary event. Please also describe how the Connector is used in this example like: Using Connectors as Boundary Events allows to receive messages while another activity is executing or waiting.

I would therefore recommend to model is with a user task which gets interrupted for example.